File: code-examples/java/orders/update.java
HttpResponse<String> response = Unirest.put("http://api.marketcloud.it/v0/orders/9")
  .header("authorization", "YourPublicKey:YourToken")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .body("{\"state\" : \"Processing\",\"total\" : 99.99, \"display_total\" : \"$ 99.99\", \"items_total\" : 69.99, \"dispaly_item_total\" : \"$ 69.99\", \"shipping_address_id\" : 123, \"billing_address_id\" : 123}")
  .asString();

File: code-examples/swift/orders/update.swift
import Foundation

let headers = [
  "authorization": "YourPublicKey:YourToken",
  "accept": "application/json",
  "content-type": "application/json"
]

let postData = NSData(data: "{"state" : "Processing","total" : 99.99, "display_total" : "$ 99.99", "items_total" : 69.99, "dispaly_item_total" : "$ 69.99", "shipping_address_id" : 123, "billing_address_id" : 123}".dataUsingEncoding(NSUTF8StringEncoding)!)

var request = NSMutableURLRequest(URL: NSURL(string: "http://api.marketcloud.it/v0/orders/9")!,
                                        cachePolicy: .UseProtocolCachePolicy,
                                    timeoutInterval: 10.0)
request.HTTPMethod = "PUT"
request.allHTTPHeaderFields = headers
request.HTTPBody = postData

let session = NSURLSession.sharedSession()
let dataTask = session.dataTaskWithRequest(request, completionHandler: { (data, response, error) -> Void in
  if (error != nil) {
    println(error)
  } else {
    let httpResponse = response as? NSHTTPURLResponse
    println(httpResponse)
  }
})

dataTask.resume()